### ISSUE TITLE
ToString of null Error

### ISSUE THREAD

**Issue by rspraveenraj** [martndemus/ember-font-awesome#209]
On version 3.1.1 of font awesome we are facing an issue with fa-con.js . When the size is not defined , the getWithDefault method throws undefined and due to that .toString of undefined throws an error.

**Comment by sherinkandiyan** [comment]
Yeah,I am facing the same issue.It is causing Qunit test cases to fail in my Ember project after upgrading Ember to 2.18.2

**Comment by sherinkandiyan** [comment]
Please suggest a solution for the above issue
